mautic/mautic
# Allow array values in api filter paramaters

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | Y
| BC breaks? | N
| Deprecations? | N 

Description:

We do not allow for API calls to use multiple arguments for anything but ids. The goal of this PR is to allow them.
Steps to test and reproduce this PR:

    Make simillar API calls with multiple arguments for something else than ids

/api/contacts?where[0][col]=global_customer_id&where[0][expr]=in&where[0][val]="10","11"
/api/contacts?where[0][col]=global_customer_id&where[0][expr]=in&where[0][val]=10,11

    They don't work
    Load up the PR
    They should work correctly now
